emr32/build.properties

```diff
@@ -0,0 +1,36 @@
+impl.title=Pentaho Hadoop Amazon EMR 3.2 Shim
+impl.productID=pentaho-hadoop-shims-emr32
+ivy.artifact.id=pentaho-hadoop-shims-emr32
+ivy.artifact.group=pentaho
+project.revision=TRUNK-SNAPSHOT
+package.id=pentaho-hadoop-shims-emr32-package
+package.root.dir=emr32
+
+dependency.kettle.revision=TRUNK-SNAPSHOT
+dependency.pentaho-hdfs-vfs.revision=TRUNK-SNAPSHOT
+dependency.pentaho-hadoop-snappy.revision=0.0.1
+dependency.pentaho-vfs-browser.revision=TRUNK-SNAPSHOT
+
+dependency.xstream.revision=1.4.2
+dependency.commons-lang.revision=2.6
+dependency.jackson.revision=2.5.1
+jackson.version=2.5.1
+dependency.amazon-sdk.revision=1.9.18
```

Comment:
It doesn't appear this version of the Amazon SDK has been approved via the Component Process